src/components/ui/TimePicker/index.tsx

```diff
@@ -63,7 +73,7 @@ export default function TimePicker({ value, onChange }: TimePickerProps) {
           side="bottom"
           align="end"
           sideOffset={4}
-          className="w-32 overflow-hidden rounded-lg border border-gray-500 bg-gray-700"
+          className="w-(--radix-popover-trigger-width) overflow-hidden rounded-lg border border-gray-500 bg-gray-700"
         >
           <div className="h-56 overflow-y-auto">
```

Comment:
```suggestion
          <div className="h-56 overflow-y-auto [&::-webkit-scrollbar]:w-1.5 [&::-webkit-scrollbar-thumb]:rounded-full [&::-webkit-scrollbar-thumb]:bg-gray-500 [&::-webkit-scrollbar-track]:bg-transparent">
```

스크롤바 관련 속성 추가해주시면 좋을 것 같습니다

적용 전 | 적용 후
---|---
<img width="131" height="306" alt="Image" src="https://github.com/user-attachments/assets/c952abb1-5395-45a0-bf5a-bc665bccd86e" />|<img width="133" height="305" alt="Image" src="https://github.com/user-attachments/assets/2c7962a8-56d1-4688-a80c-cee171b0adcc" />